From cf9a4b070ccb4a7a81c4d34389e626436d55c643 Mon Sep 17 00:00:00 2001
From: Joel Capitao <jcapitao@redhat.com>
Date: Wed, 13 Sep 2023 09:22:56 +0200
Subject: [PATCH] Remove obsolete whitelist_externals

allowlist_externals was added in [1] but the obsolete
whitelist_external was not removed.

[1] https://review.opendev.org/c/openstack/python-saharaclient/+/893329

Change-Id: I1c747fee8270db729227ca0e882b93ad1812c81f
---
 tox.ini | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tox.ini b/tox.ini
index 78537b1..f23f88e 100644
--- a/tox.ini
+++ b/tox.ini
@@ -19,8 +19,6 @@ deps =
     -r{toxinidir}/test-requirements.txt
 commands = find . -type f -name "*.pyc" -delete
            stestr run {posargs}
-whitelist_externals = find
-                      rm
 allowlist_externals = find
                       rm
 passenv =
-- 
2.41.0

